From 2472a9d281490601cb142e356464a4ee3e705f2e Mon Sep 17 00:00:00 2001
From: Jacky Chou <jacky_chou@aspeedtech.com>
Date: Thu, 7 May 2026 17:45:20 +0800
Subject: [PATCH] board: aspeed: evb-ast2705: add MAC boot command

Add a bootmac environment command that uses DHCP and boots the
loaded image. This gives the EVB config a reusable command for MAC
recovery boot.

Upstream-Status: Backport [SDK v11.03 migration]
Signed-off-by: Jacky Chou <jacky_chou@aspeedtech.com>
Change-Id: Ifdd0953f1dfc575809cbcd4206b8e736d67ce8d7
Signed-off-by: rupeshs <rupeshs@ami.com>
---
 include/configs/evb_ast2705.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/evb_ast2705.h b/include/configs/evb_ast2705.h
index 764daf4544..5925068091 100644
--- a/include/configs/evb_ast2705.h
+++ b/include/configs/evb_ast2705.h
@@ -30,5 +30,6 @@
 	"ufsbootb=setenv bootpart 3; setenv rootfs rofs-b; run setufsargs; ext4load scsi 0:${bootpart} ${loadaddr} fitImage && bootm ${loadaddr}; echo Error loading kernel FIT image\0"	\
 	"bootufs=if test \"${bootside}\" = \"b\"; then run ufsbootb; run ufsboota; else run ufsboota; run ufsbootb; fi\0"	\
 	"verify=no\0"	\
+	"bootmac=dhcp && bootm\0"	\
 	""
 #endif	/* __CONFIG_H */
-- 
2.43.0

